server/src/services/follows.js

```diff
@@ -1,13 +1,23 @@
 const Follows = require("../db/mongo/models/follow");
 
-const searchFollowings = async (email) => {
-    return Follows.find({ email_from: email}).exec();
+const searchFollowings = async (email, pageNumber, pageLimit) => {
+    const totalAmount = (await Follows.find({ email_from: email}).exec()).length;
```

Comment:
```
const aggregationQuery = [  
    { $match: { email_from: email }} 
];
const data = await Follows.aggregate([...aggregationQuery, ...paginationQuery(pageNumber, pageLimit)]);
const dataTotalAmount = await Follows.aggregate([...aggregationQuery, { $count: "count" }]);
const totalAmount = dataTotalAmount[0] ? dataTotalAmount[0].count : 0;

return { data, totalAmount };
```
Similar to what Shuker did in series. Apply something similar in `searchFollowers`